tests/MorseCodeTest.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+use function PHPUnit\Framework\assertEquals;
+use PHPUnit\Framework\TestCase;
+
+require_once __DIR__ . '/../vendor/autoload.php';
+require_once __DIR__ . '/../ciphers/morseCode.php';
+
+class CiphersTest extends TestCase
```

Comment:
`CiphersTest` already exists [here](https://github.com/TheAlgorithms/PHP/blob/master/tests/CiphersTest.php)

```suggestion
class MorseCodeTest extends TestCase
```